mackup/applications/i3wm.cfg

```diff
@@ -0,0 +1,7 @@
+[application]
+name=i3wm
+
+[configuration_files]
+.i3/config
+.i3status.conf
+.xsession
```

Comment:
We can't have configs leak from one program to another, or every program will have all the configs. It should be in the X11 config.